### PR TITLE
DOC: Fix some docstrings in filter.py

### PR DESCRIPTION
**Pull request by TomDonoghue**
#### What does this implement/fix?
Fixes some small docstring errors in the filter.py file. 

As I was looking through `filter.py`, I realized there were some small mistakes in the docstrings of a couple functions (for examples, `sfreq` being described as `Fs` in the docstring), so I figured I could make a quick fix on those.

### CODE REVIEW

File: mne/filter.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def is_power2(num):
 
     Returns
     -------
-    b : bool
+    bool
```
Comment (agramfort) [inline mne/filter.py]: Are you sure about this one? We always name our returned arguments

Comment (TomDonoghue) [reply mne/filter.py]: Ah, maybe I have mis-understood the convention here. In the code (which I didn't change any of), the return is anonymous - the literal line is: `return num != 0 and ((num & (num - 1)) == 0)`. I thought in that case the docstring should be un-named, but based on Eric's comment it seems like MNE convention is give a reasonable name in the docstring, even if it's an unnamed return in the code? If so, I can switch this back!

Comment (agramfort) [reply mne/filter.py]: yes please switch it back. We always give names to returned values  thx

Comment (larsoner) [reply mne/filter.py]: Yes we always name even if anonymous. And even when not anonymous the name we give the user is not necessarily the same as the name in the implementation `return <name>`. So far we mostly try to think of an informative return name.

Comment (TomDonoghue) [reply mne/filter.py]: I see, that makes sense, thank you for notes on this! I have switched that change back to how it was!

**Comment by larsoner** [comment]
Thanks @TomDonoghue

**Pull request merged**